### PR TITLE
Update postgres role and config for db integration

### PR DESCRIPTION
**Pull request by mkllnk**
The db_integrations playbook was broken. I guess that the internal workings of the postgresql role changed and it didn't expect a subtask to be included on its own. I'm now including the whole role as in the provisioning playbook to make it work again and reduce maintenance. It will be a little bit slower though.

### CODE REVIEW

**Review by jibees — APPROVED** [review]

**Comment by jibees** [comment]
Once this is merged, don't know how the process continue: should we provision all the servers?

**Comment by mkllnk** [comment]
> should we provision all the servers?

No, this is just a fix to make sure that we can run the playbooks when needed. But it doesn't need to run now. It will unblock the n8n user provisioning.

**Pull request merged**
